content/blog/morpheus-plugin-tutorial-how-to-build-and-compile.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "Morpheus plugin tutorial: How to build and compile"
+date: 2025-11-19T08:43:36.028Z
+author: Neil van Rensburg
+authorimage: /img/greenlogo.png
+disable: false
+---
+## Introduction
+
+This article deals with generating and compiling a basic Morpheus plugin project on Windows 11.
+
+Topics covered in this article:
+
+* How to generate a new project using the plugin code generator
+* Unzip and open the project in an IDE
+* Explore main plugin file components
+* Compile the plugin on Windows
+* Upload the compiled plugin to Morpheus
+* Compile the plugin remotely on Linux, using Visual Studio Code
+* Compile the plugin using docker
+
+## JDK Prerequisite
+
+For the labs in this document, we assume a **Windows 11 host** with internet access and **Visual Studio Code** installed.
```

Comment:
One thing I'm kind of a stickler for is eliminating the use of the "Royal We". I know it's actually taught in engineering schools, but from a reader's perspective (in the "outside world"), it can sometimes get confusing, especially since we have a broad audience worldwide with many who may not have English as their first language.

So, I'll make some recommendations on what you can do instead.  Here, I would say:

"For the labs in this document, it is assumed that there is a **Windows 11 host** with internet access and that **Visual Studio Code** is installed."